BootloaderCorePkg/Library/FspSupportLib/FspSupportLib.c

```diff
@@ -114,3 +114,43 @@ TraverseMemoryResourceHob (
     Hob.Raw = GET_NEXT_HOB (Hob);
   }
 }
+
+
+/**
+  Dump FSP memory resource
+
+  @param  HobListPtr         A HOB list pointer.
+
+**/
+VOID
+EFIAPI
+DumpFspResourceHob (
+  IN  CONST VOID            *HobListPtr
+  )
+{
+  EFI_PEI_HOB_POINTERS    Hob;
+
+  /*
+   * Get the HOB list for processing
+   */

```

Comment:
Not prefer using /* */ as the comments. same for other place.

Reply:
will change